finetune/target_models/masked_language_model.py

```diff
@@ -142,6 +142,20 @@ def finetune(self, X, batch_size=None, **kwargs):
         """
         return super().finetune(X, Y=None, batch_size=batch_size, **kwargs)
 
+    def featurize(self, X, context=None):
+        hold_mask_proba = self.config.mask_proba
+        self.config.mask_proba = 0.0
+        output = super().featurize(X, context=context)
+        self.config.mask_proba = hold_mask_proba
+        return output
+
+    def	featurize_sequence(self, X, context=None):
```

Comment:
...

Reply:
you've just got tabs and spaces everywhere haha.  PR looks good though

Reply:
I can't see any more in this PR??